src/main/webapp/app/overview/student-questions/student-question-row.component.html

```diff
@@ -135,13 +195,52 @@ <h6 class="text-black-50 question-subtitle">{{ studentQuestion.author.name }} -
                     </div>
                 </div>
             </div>
+            <!-- unapproved answers -->
```

Comment:
The code below looks very similar to the one for the approved answers. There also seems to be a lot of shared template code between the view and edit state (see screenshot below).

I think it would make sense to avoid this duplication by creating small new components whenever reusable parts like these manifest themselves.

Code duplicate:
![OnPaste 20200421-121427](https://user-images.githubusercontent.com/7109225/79854428-b6fe3680-83c9-11ea-9bc7-04794361b381.png)

Reply:
I will address the code duplication (or at least very similar code) in a follow-up PR, in which I change the layout of the questions and answers (and also refactor them into smaller components).
@HanyaElhashemy

Reply:
Great, thanks!